### ISSUE TITLE
Have you achieve the accuracy that reported in original paper?

### ISSUE THREAD

**Issue by a14en9** [kdkalvik/ST-GCN#1]
Hi,

Many thanks for sharing the code based on Tensorflow.

Just a quick question.

How much accuracy have you got? I can only achieve 71% without shuffling the training datasets. If I set the shuffle as True, it will throw the error: "filling up shuffle buffer (this may take a while)". Have you faced the same question and any idea can be used to solve it please?

Thanks a lot

**Comment by itskalvik** [comment]
Yup, had the same issue with the buffer warning. It's just a warning, if you wait a while, it will eventually fill the buffer and start training. You can try decreasing the buffer size, that will speed it up. But, make sure the dataset is pre shuffled when generating the tfrecords. Also, I was able to get the results reported in the original paper, I made the pretrained weights publicly available, check the readme.

**Comment by a14en9** [comment]
Hi, many appreciate your prompt reply. I'll have a look then.

Thanks a lot.

**Comment by a14en9** [comment]
> Yup, had the same issue with the buffer warning. It's just a warning, if you wait a while, it will eventually fill the buffer and start training. You can try decreasing the buffer size, that will speed it up. But, make sure the dataset is pre shuffled when generating the tfrecords. Also, I was able to get the results reported in the original paper, I made the pretrained weights publicly available, check the readme.

Hi, my apologies to disturb you again.  Do you have any idea to visualise the prediction results on the skeleton, please?

**Comment by itskalvik** [comment]
You can try using matplotlib or you could try ROS. The new Azure Kinect camera cones with a ROS package, so you could stream that data to this classifier. But, I don't plan on adding such implementations here.

**Comment by itskalvik** [comment]
BTW, I forgot to mention this before, the training logs for my results are in the logs folder. You can use tensorboard to see the results I got.

**Comment by a14en9** [comment]
Yup, I can train my own networks now. thanks a lot.
